packages/api/src/server.ts

```diff
@@ -45,6 +45,12 @@ export class Server {
     }
 
     app.addHook("preHandler", (req, reply, done) => {
+      
+      if (req.url === "/version" || req.url === "/healthcheck") {
```

Comment:
Pls refactor code

```
if (req.url === "/version" || req.url === "/healthcheck" ) {
    // do nothing
} else if (req.method == "POST") {
   //existing code
} else {
   // existing code
}
done()

```

Reply:
yes @ch4r10t33r , tested and it is working fine. I don't see neither /version nor /healthcheck being logged.
<img width="337" alt="image" src="https://github.com/user-attachments/assets/6bed62ae-4830-4fe0-b9d2-6c3625150ae7">

Reply:
Please merge after testing.